src/Platforms/PostgreSQLPlatform.php

```diff
@@ -833,6 +833,16 @@ public function getDropIndexSQL($index, $table = null)
             return $this->getDropConstraintSQL($constraintName, $table);
         }
 
+        if ($table !== null) {
+            $indexName = $index instanceof Index ? $index->getQuotedName($this) : $index;
+            $tableName = $table instanceof Table ? $table->getQuotedName($this) : $table;
+
+            if (strpos($tableName, '.') !== false) {
+                [$schema] = explode('.', $tableName);
+                $index    = $schema . '.' . $indexName;
```

Comment:
The functional test would fail if these 2 lines were not called. Possible an false positive because it's a new test file added in this PR?

Reply:
Yeah, looks like false positive (the [diff coverage](https://app.codecov.io/gh/doctrine/dbal/pull/6779?dropdown=coverage&src=pr&el=h1) is 100%).